src/utils/share/index.js

```diff
@@ -59,10 +59,10 @@ const shareFeeds = async (item, analyticsLogEvent, analyticsId) => {
   await buildShare(`${config.POST_SHARE_URL}/post/${item?.id}`);
 };
 
-const shareUserLink = (username) => buildShare(`${config.SHARE_URL}/u/${username}`);
+const shareUserLink = (username) => buildShare(`${config.SHARE_URL}/${username}`);
 
 const copyToClipboard = (username) => {
-  Clipboard.setString(`${config.SHARE_URL}/u/${username}`);
+  Clipboard.setString(`${config.SHARE_URL}/${username}`);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the URL structure has been changed from `/u/` to `/`. Make sure this change is consistent across your application and backend to avoid any broken links.
```diff
- const shareUserLink = (username) => buildShare(`${config.SHARE_URL}/u/${username}`);
- const copyToClipboard = (username) => {
-   Clipboard.setString(`${config.SHARE_URL}/u/${username}`);
- };
+ const shareUserLink = (username) => buildShare(`${config.SHARE_URL}/${username}`);
+ const copyToClipboard = (username) => {
+   Clipboard.setString(`${config.SHARE_URL}/${username}`);
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->